microprofile/microprofile-rest-client-1.3/src/main/java/org/wildfly/swarm/ts/microprofile/rest/client/v13/RestClientResource.java

```diff
@@ -70,4 +77,42 @@ public String close() throws Exception {
         }
         return response;
     }
+
+    @GET
+    @Path("/ssl-client")
+    public String correctClient() {
+
+        ClassLoader contextClassLoader = Thread.currentThread()
+                .getContextClassLoader();
+        InputStream truststoreInputStream = contextClassLoader.getResourceAsStream("/META-INF/client-truststore.jks");
+        InputStream keystoreInputStream = contextClassLoader.getResourceAsStream("/META-INF/client-keystore.jks");
+
+        try {
+            KeyStore trustStore = getKeyStore(truststoreInputStream);
+            KeyStore keyStore = getKeyStore(keystoreInputStream);
+
+
+            return RestClientBuilder.newBuilder()
+                    .baseUrl(new URL(HTTP_LOCALHOST_8080))
+                    .trustStore(trustStore)
+                    .keyStore(keyStore, PASSWORD)
+                    .hostnameVerifier(new AcceptAllVerifier())
+                    .build(SslClient.class).simple();
```

Comment:
The call to `.simple()` should be on an extra line, it's the most important thing that happens here.